templates/managed/kafka.html

```diff
@@ -0,0 +1,303 @@
+        <td colspan="2">Yes, support in deploying your organisation's Schema Registry</td>
+      </tr>
+      <tr>
+        <td><strong>Metrics and Visualisation</strong></td>
+        <td colspan="2">Yes, Managed Kafka includes Graylog, Prometheus and Grafana monitoring and dashboard tools, to ensure transparency and visibility on all vital metrics</td>
+      </tr>
+    </table>
+
+    <h3>Interoperability</h3>
+    <table>
+      <tr>
+        <td><strong>Connectors</strong></td>
+        <td colspan="2">Yes, open source Kafka Connectors for applications</td>
+      </tr>
+      <tr>
+        <td><strong>REST API</strong></td>
+        <td colspan="2">Yes, administrate Kafka without accessing the underlying Kafka client</td>
+      </tr>
+      <tr>
+        <td><strong>Direct</strong></td>
+        <td colspan="2">Hostname and IP visibility, to connect with your other cloud infrastructure</td>
+      </tr>
+    </table>
+
+    <h3>Security and reliability</h3>
+    <table>
+      <tr>
+        <td><strong>Security Patches</strong></td>
+        <td colspan="2">Yes. Canonical provides all security patches.</td>
+      </tr>
+      <tr>
+        <td><strong>Kernel</strong></td>
+        <td colspan="2">Kernel Livepatch</td>
+      </tr>
+      <tr>
+        <td><strong>24/7 Monitoring</strong></td>
+        <td colspan="2">Yes &mdash; Canonical&rsquo;s Kafka experts cover all geographic regions and time-zones</td>
+      </tr>
+      <tr>
+        <td><strong>Break/Fix response</strong></td>
+        <td colspan="2">Yes</td>
+      </tr>
+      <tr>
+        <td><strong>SLA</strong></td>
+        <td colspan="2">Yes, 99.9% up-time.</td>
+      </tr>
+      <tr>
+        <td><strong>ISO270001/2 certified</strong></td>
+        <td colspan="2">Yes</td>
+      </tr>
+      <tr>
+        <td><strong>SOC2 Type 2 certified</strong></td>
+        <td colspan="2">Yes</td>
+      </tr>
+      <tr>
+        <td><strong>GDPR Compliance tested</strong></td>
+        <td colspan="2">Yes</td>
+      </tr>
+    </table>
+
+    <p><a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Contact us</a></p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-equal-height">
+    <div class="col-7">
+      <h2>Learn how Managed Apps help organisations maintain business focus</h2>
+      <p>Managed Apps frees DevOps teams to focus on the value their apps can deliver and away from time-consuming cloud-management tasks, at a predictable price.</p>
+      <p><a class="p-button--positive" href="https://www.brighttalk.com/webcast/6793/394568">Watch on-demand</a></p>
+    </div>
+    <div class="col-5 u-vertically-center">
+      <blockquote class="p-pull-quote">
+        <p class="p-pull-quote__quote">With Managed Apps, DevOps becomes a strategic edge, not just a requirement for others to perform their jobs.</p>
+        <cite class="p-pull-quote__citation">Canonical</cite>
```

Comment:
+1 except this line... the copy doc doesn't have a citation and I think it is odd to use Canonical... can you check that this is meant to be here? I see it in the design, but no where else.